pulp.spec

```diff
@@ -325,6 +325,7 @@ touch %{buildroot}/%{_sysconfdir}/pki/%{name}/consumer/server/rsa_pub.key
 cp -R agent/etc/pulp/agent/agent.conf %{buildroot}/%{_sysconfdir}/%{name}/agent/
 cp -R client_consumer/etc/pulp/consumer/consumer.conf %{buildroot}/%{_sysconfdir}/%{name}/consumer/
 %if 0%{?rhel} >= 6 || 0%{?fedora} >= 19
+mkdir -p %{buildroot}/%{_sysconfdir}/bash_completion.d
```

Comment:
Without it, this directory is only created if pulp-admin is also being built. So the conditional on line 4 combined with the conditional on line 327 combine to break the build, unless this mkdir call is here.

Beyond that, I think it's just good practice to be very explicit in packaging, which includes always ensuring a destination dir exists before trying to copy files to it, whether or not any other package is supposed to have created it, particularly when no special permissions are involved.